live-examples/js-examples/string/string-split.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = "The quick brown fox jumped over the lazy dog.";
```

Comment:
Please use single quotes for strings.

Reply:
Since `split()` is not ES6, please don't use ES6 constructs like `const`: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6.